### PR TITLE
doc: add minutes for meeting 23 Feb

### PR DESCRIPTION
**Pull request by mhdawson**
Signed-off-by: Michael Dawson <mdawson@devrus.com>

### CODE REVIEW

**Review by dominykas — APPROVED** [review]

File: meetings/2021-02-23.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+* Suggest ignoring a vulnerability by the package maintainer [#386](https://github.com/nodejs/package-maintenance/issues/386)
+  * Collaboration space approved, Michael worked with Darcy to set up repo - https://github.com/openjs-foundation/pkg-vuln-collab-space
+
+* Next steps on Support levels in Package.json [#192](https://github.com/nodejs/package-maintenance/issues/192)
+  * https://developers.redhat.com/blog/2021/02/10/add-standardized-support-information-to-your-node-js-modules/
+  * Landed the “create” command pkg support module - https://github.com/pkgjs/support/pull/40
+  * Dominikas, will see if NearForm will add to their modules
+  * Michael will follow up with npm in terms of showing info on the npm UI.
+
+* Discussion about Wibi
```
Comment (BethGriggs) [inline meetings/2021-02-23.md:46]:
```suggestion
* Discussion about Wiby
```

**Review by BethGriggs — APPROVED** [review]

File: meetings/2021-02-23.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# Node.js  Package Maintenance Team Meeting 2021-02-23
+
+## Links
+
+* **Recording**: https://studio.youtube.com/video/aaH1XS_QBNM/edit
```
Comment (rodion-arr) [inline meetings/2021-02-23.md:5]:
```suggestion
* **Recording**: https://www.youtube.com/watch?v=aaH1XS_QBNM
```

File: meetings/2021-02-23.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+## Agenda
+
+## Announcements
+ 
+*Extracted from **package-maintenance-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/package-maintenance
+
+* OpenJS World 2021 CFP [#447](https://github.com/nodejs/package-maintenance/issues/447)
+  * Glen submitted the panel submission
```
Comment (rodion-arr) [inline meetings/2021-02-23.md:26]:
```suggestion
  * Glen submitted the panel discussion
```

**Review by rodion-arr — APPROVED** [review]

**Comment by rodion-arr** [comment]
@dominykas, please tag me on wiby tasks also, will be glad to help

**Review by thescientist13 — APPROVED** [review]

**Review by ghinks — APPROVED** [review]

**Pull request merged**